### PR TITLE
Fix/update migrations

### PR DESCRIPTION
**Pull request by a-kuharenko**

### CODE REVIEW

File: server/data/models/star.js [hunk 1]
```diff
@@ -2,10 +2,16 @@ module.exports = (sequelize, DataTypes) => {
   const Star = sequelize.define(
     'star',
     {
+      userId: DataTypes.UUID,
+      repositoryId: DataTypes.UUID,
```
Comment (NickNaumenko) [inline server/data/models/star.js]: "When you create associations between your models in sequelize, foreign key references with constraints will automatically be created". https://sequelize.org/master/manual/associations.html#foreign-keys
тоже самое касается и других моделей

File: server/data/migrations/20190823091110-create-tables.js [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+        type: Sequelize.STRING,
+        unique: false
+      },
+      type: {
+        allowNull: false,
+        type: Sequelize.TEXT,
+        unique: false,
+        defaultValue: 'USER'
+      },
+      fake: {
```
Comment (OlgaGnatenko) [inline server/data/migrations/20190823091110-create-tables.js]: What is a fake user? If it is "organization", then we already have "type" above

File: server/data/migrations/20190823091110-create-tables.js [hunk 3]
```diff
@@ -0,0 +1,358 @@
...
+        allowNull: false,
+        type: Sequelize.UUID,
+        references: {
+          model: 'users',
+          key: 'id'
+        }
+      },
+      description: {
+        type: Sequelize.TEXT,
+        unique: false
```
Comment (OlgaGnatenko) [inline server/data/migrations/20190823091110-create-tables.js]: doesn't unique equal to false by default?

File: server/data/migrations/20190823091110-create-tables.js (hunk 2 above)
Comment (AlikRakhmonov) [reply server/data/migrations/20190823091110-create-tables.js]: fixed

File: server/data/migrations/20190823091110-create-tables.js (hunk 3 above)
Comment (AlikRakhmonov) [reply server/data/migrations/20190823091110-create-tables.js]: fixed

**Pull request merged**